### ISSUE TITLE
Ctrl+a brings an error

### ISSUE THREAD

**Issue by VPenkov** [avocode/avocode#404]
I created an export rule for an asset. I decided to edit its name and pressed ctrl+a while editing it to select it all but got an error.
-----
App v.2.4.2-d618096
Source UID: 45217
```
TypeError: undefined is not a function
  at HTMLDocument.<anonymous> (C:\Users\Vergil\AppData\Local\avocode\app-2.4.2\resources\app.asar\src\window-event-handler.js:170:20)
  at HTMLDocument.handler (C:\Users\Vergil\AppData\Local\avocode\app-2.4.2\resources\app.asar\src\space-pen-extensions.js:115:32)
  at HTMLDocument.jQuery.event.dispatch (C:\Users\Vergil\AppData\Local\avocode\app-2.4.2\resources\app.asar\node_modules\space-pen\vendor\jquery.js:4681:9)
  at HTMLDocument.elemData.handle (C:\Users\Vergil\AppData\Local\avocode\app-2.4.2\resources\app.asar\node_modules\space-pen\vendor\jquery.js:4359:46)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (C:\Users\Vergil\AppData\Local\avocode\app-2.4.2\resources\app.asar\src\command-registry.js:251:29)
  at C:\Users\Vergil\AppData\Local\avocode\app-2.4.2\resources\app.asar\src\command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (C:\Users\Vergil\AppData\Local\avocode\app-2.4.2\resources\app.asar\node_modules\atom-keymap\lib\keymap-manager.js:520:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (C:\Users\Vergil\AppData\Local\avocode\app-2.4.2\resources\app.asar\node_modules\atom-keymap\lib\keymap-manager.js:355:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (C:\Users\Vergil\AppData\Local\avocode\app-2.4.2\resources\app.asar\src\window-event-handler.js:184:27)

```

**Comment by rychlis** [comment]
thanks for the report, it's a bug in atom package we are using and only happens on Windows and Linux. It has been [fixed in atom](https://github.com/atom/atom/issues/6587) some time ago but we are using older unfixed version at the moment

**Comment by VPenkov** [comment]
Might wanna change that milestone. It says 2.4.3 but it's still an existing issue.

**Comment by rychlis** [comment]
true. We are now in process of removing atom completely in order to run on clear electron, which should also fix this issue (and many others)

**Comment by petrbrzek** [comment]
Fixed in 2.8. Closing now.
